modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -4301,14 +4301,20 @@ Edit science requirements
 """
 input ScienceRequirementsInput {
   """
-  The mode field must be either specified or skipped altogether.  It cannot be unset with a null value.
+  ExposureTimeMode, which may be unset by assigning a null value, or ignored by
+  skipping it altogether.
   """
-  mode: ScienceMode
```

Comment:
`mode` is removed, this matches other inputs. We don't set `mode` instead it is inferred depending on whether `spectroscopy` or `imaging` is set